BuildResidentialHPXML/measure.rb

```diff
@@ -1219,6 +1219,60 @@ def arguments(model)
     arg.setDescription('The conditioned floor area served by the air distribution system.')
     args << arg
 
+    heating_system_type_2_choices = OpenStudio::StringVector.new
+    heating_system_type_2_choices << 'none'
+    heating_system_type_2_choices << HPXML::HVACTypeWallFurnace
+    heating_system_type_2_choices << HPXML::HVACTypeFloorFurnace
+    heating_system_type_2_choices << HPXML::HVACTypeElectricResistance
+    heating_system_type_2_choices << HPXML::HVACTypeStove
+    heating_system_type_2_choices << HPXML::HVACTypePortableHeater
+    heating_system_type_2_choices << HPXML::HVACTypeFireplace
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('heating_system_type_2', heating_system_type_2_choices, true)
+    arg.setDisplayName('Heating System 2: Type')
+    arg.setDescription('The type of the second heating system.')
+    arg.setDefaultValue('none')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('heating_system_fuel_2', heating_system_fuel_choices, true)
+    arg.setDisplayName('Heating System 2: Fuel Type')
+    arg.setDescription('The fuel type of the second heating system. Ignored for ElectricResistance.')
+    arg.setDefaultValue(HPXML::FuelTypeElectricity)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_heating_efficiency_afue_2', true)
+    arg.setDisplayName('Heating System 2: Rated AFUE')
+    arg.setUnits('AFUE')
+    arg.setDescription('The rated efficiency value of the second Furnace/WallFurnace heating system.')
+    arg.setDefaultValue(0.78)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('heating_system_heating_efficiency_percent_2', true)
+    arg.setDisplayName('Heating System 2: Rated Percent')
+    arg.setUnits('Percent')
+    arg.setDescription('The rated efficiency value of the second ElectricResistance/Stove/PortableHeater/Fireplace heating system.')
+    arg.setDefaultValue(1.0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument('heating_system_heating_capacity_2', true)
+    arg.setDisplayName('Heating System 2: Heating Capacity')
+    arg.setDescription("The output heating capacity of the second heating system. If using '#{Constants.Auto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
```

Comment:
I suggest rewording to "the autosizing algorithm will use ACCA Manual J/S to set the capacity to meet its load served." That will help a user understand that a system meeting 50% of the heating load will be autosized to 50% of the capacity it would have if it met 100% of the heating load. Make the same change for the primary HVAC system.